build.gradle

```diff
@@ -60,6 +60,10 @@ subprojects {
         options.noTimestamp()
     }
 
+    tasks.withType(JavaCompile).configureEach {
+        options.compilerArgs << '-parameters'
```

Comment:
@loowps thanks for your research and the PR, but could you please link to the source docs (gradle/jvm?) to show that this arg disables the implicit '-debug' arg?

Reply:
@bartoszwalacik I used https://github.com/gradle/gradle/issues/14781 as a reference for the configuration change.

What I did manually to verify the solution:
- create a spring-boot maven project with Spring Data MongoDB dependency via https://start.spring.io/
-add to the pom.xml
```
<dependency>
    <groupId>org.javers</groupId>
    <artifactId>javers-spring-boot-starter-mongo</artifactId>
    <version>7.3.4</version>
</dependency>
```

which leads to the following warning on application startup:
```
WARN 6296 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.javers.spring.auditable.aspect.JaversAuditableAspect
WARN 6296 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.javers.spring.auditable.aspect.springdata.JaversSpringDataAuditableRepositoryAspect
WARN 6296 --- [           main] ocalVariableTableParameterNameDiscoverer : Using deprecated '-debug' fallback for parameter name resolution. Compile the affected code with '-parameters' instead or avoid its introspection: org.javers.spring.auditable.aspect.springdata.JaversSpringDataAuditableRepositoryAspect
```

- build and publish javers with my PR changes to my local maven repository (change version to 7.3.5 and disable signing)
- updated the dependency in the test project, reimported the maven project and started the application
=> the 3 warnings are gone